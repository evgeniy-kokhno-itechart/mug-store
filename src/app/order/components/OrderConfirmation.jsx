import React from 'react';
import { ToCatalogButton } from '../../catalog';

const OrderConfirmation = () => (
  <React.Fragment>
    <h1 className="text-center m-3">Thank you for your order!</h1>
    <p className="h3 text-center">Our operator will call you shortly</p>

    <div className="d-flex">
      <ToCatalogButton customClasses="mt-3 mx-auto w-50" />
    </div>
  </React.Fragment>
);

export default OrderConfirmation;
