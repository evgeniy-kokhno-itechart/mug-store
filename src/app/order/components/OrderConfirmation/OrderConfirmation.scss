.order-confirm {
    &__header {
        text-align: center;
        margin: 1.5rem;
    }

    &__message {
        font-size: 2rem;
        font-weight: 500;
        text-align: center;
    }

    &__button {
        display: block;
        width: 45%;
        margin: 0 auto;
    }
}