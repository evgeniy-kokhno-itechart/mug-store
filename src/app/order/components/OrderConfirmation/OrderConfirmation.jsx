import React from 'react';
import { ToCatalogButton } from '../../../catalog';
import './OrderConfirmation.scss';

const OrderConfirmation = () => (
  <div className="order-confirm">
    <h1 className="order-confirm__header">Thank you for your order!</h1>
    <p className="order-confirm__message">Our operator will call you shortly</p>
    <ToCatalogButton customClasses="order-confirm__button" />
  </div>
);

export default OrderConfirmation;
