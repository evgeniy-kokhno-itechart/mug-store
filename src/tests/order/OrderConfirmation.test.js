/* eslint-disable no-undef */
import React from 'react';

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

import OrderConfirmation from '../../order/components/OrderConfirmation';

configure({ adapter: new Adapter() });

describe('<OrderConfirmation />', () => {
  let OrderConfirmationWrapper;

  beforeEach(() => {
    OrderConfirmationWrapper = shallow(<OrderConfirmation />);
  });

  it('check OrderConfirmation renders properly', () => {
    expect(OrderConfirmationWrapper.debug()).toMatchSnapshot();
  });
});
