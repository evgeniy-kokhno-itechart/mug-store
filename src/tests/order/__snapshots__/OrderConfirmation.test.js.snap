// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrderConfirmation /> check OrderConfirmation renders properly 1`] = `
"<Fragment>
  <h1 className=\\"text-center m-3\\">
    Thank you for your order!
    <br />
    <br />
    Our operator will call you shortly.
  </h1>
  <div className=\\"d-flex\\">
    <ToCatalogButton customClasses=\\"mt-3 mx-auto w-50\\" />
  </div>
</Fragment>"
`;
